app/styles/app.scss

```diff
@@ -801,6 +801,72 @@ body::-webkit-scrollbar-thumb {
   box-shadow: none;
   @apply border-4 border-dotted border-df-green;
 }
+#mobile-navbar {
+  max-height: 100vh;
+  margin: 0;
+  padding: 12px;
+
+  div:first-child {
+    height: 100%;
+  }
+
+  button {
+    width: 1em;
+  }
+
+  #navigation {
+    max-height: calc(100% - 2em);
+    max-width: 100vw;
+    overflow-y: scroll;
+  }
+
+  @media only screen and (max-width: 600px) {
+    padding: 0 12px;
+  }
+}
+
+.podcast-info {
+  gap:0.25rem;
+}
+
+.podcast {
+  max-width: 100%;
+}
+
+#podcast-search-results {
+
+  @media only screen and (max-width: 600px) {
```

Comment:
Ideally we could do these media queries with tailwind, just because I'm lazy and it's easier to use their predefined sizes instead of having to remember the exact pixel width.
But if this makes the experience better on mobile I'll take it!
Can always refactor later (what i always tell myself haha >_>)